config/env_spec_request.go

```diff
@@ -350,11 +352,11 @@ func (e *EnvironmentSpecRequest) JWTAuthentications() []*JWTAuthentication {
 }
 
 // looks up the JWTAuthentication by name and runs verification
-// returns true if found and verified
+// returns nil if found and verified
 // any error can be in e.jwtResults[name]
-func (e *EnvironmentSpecRequest) verifyJWTAuthentication(name string) bool {
+func (e *EnvironmentSpecRequest) verifyJWTAuthentication(name string) *fault.AdapterFault {
```

Comment:
My intention was for this (and similar return values) to be returned as `error`. Although we do not intend to use the `Error()` function from the `error` interface directly, implementing it would nonetheless allow more conventional Go here, I think.

Reply:
+1 to using as `error`

Reply:
I had used error initially but changed it later on due to misunderstanding an earlier comment. Changing it back to `error`. Thanks!